rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1322,9 +1322,9 @@ def test_upgrade_db_36_to_37(user_data_dir):  # pylint: disable=unused-argument
     new_history_events = cursor.execute('SELECT * FROM history_events;').fetchall()
     for idx, entry in enumerate(old_history_events):
         if entry[11] is None:
-            assert entry[0:11] + ('none',) + entry[12:] == new_history_events[idx]
+            assert entry[0:11] + ('none',) == new_history_events[idx]
         else:
-            assert entry == new_history_events[idx]
+            assert entry[:12] == new_history_events[idx]
```

Comment:
I still think the DB upgrade is broken here.

You either move over all stuff (which is the preferred methodology) or you say you remove everyhing, **except** the custom events, and then expect them to be redecoded.

In both case you need a test.

Case 1: Test that data got migrated correctly. Including custom events
Case 2: Test that data got deleted properly, but that custom events ... and custom evm events have been preserved